vsc/model/vsc_ast.py

```diff
@@ -483,6 +483,9 @@ class Namespace:
     minor_version: Optional[int] = None
     """ Provides the minor version of the namespace. """
 
+    version_label: Optional[str] = str()
```

Comment:
Have we discussed what to do with the [syntax file](https://github.com/COVESA/vehicle_service_catalog/blob/master/syntax.md#key-object-namespaces) now when we at least partially can generate documentation. Keep up-to-date for now, or let it degrade until we are ready to replace it with a generated version?

Reply:
> Have we discussed what to do with the [syntax file](https://github.com/COVESA/vehicle_service_catalog/blob/master/syntax.md#key-object-namespaces) now when we at least partially can generate documentation. Keep up-to-date for now, or let it degrade until we are ready to replace it with a generated version?

The syntax file ought to be removed from the VSC repository since the IFEX documentation now provides all that information.  The VSC project can optionally refer to IFEX for the definition of the language used for the service descriptions.